runbotics/runbotics-orchestrator-ui/src/src-app/views/credentials/Credential/EditCredential/EditCredentialForm.tsx

```diff
@@ -0,0 +1,119 @@
+            .unwrap()
+            .then(() => {
+                enqueueSnackbar(translate('Credential.Form.Edit.Success', { name: credentialFormState.name }), { variant: 'success' });
+                collectionId
+                    ? dispatch(credentialsActions.fetchAllCredentialsInCollection({ resourceId: `${collectionId}/credentials/` }))
+                    : dispatch(credentialsActions.fetchAllCredentialsAccessibleInTenant());
+                closeDialog();
+            })
+            .catch(error => {
+                enqueueSnackbar(error.message, { variant: 'error' });
+            });
+    };
+
+    const handleInputChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        const { name, value } = e.target;
+
+        setCredentialFormState(prevState => ({
+            ...prevState,
+            [name]: value
+        }));
+
+        if (name === 'name') {
+            value.trim() === '' ? setIsFormValid(false) : setIsFormValid(true);
+        }
+    };
+
+    return (
+        <If condition={isOpen}>
+            <CustomDialog
+                isOpen={true}
+                onClose={closeDialog}
+                title={translate('Credential.Dialog.Modify.Edit.Title')}
+                confirmButtonOptions={{
+                    label: translate('Common.Save'),
+                    onClick: handleSubmit
+                }}
+                cancelButtonOptions={{
+                    onClick: closeDialog
+                }}
+            >
+                <Content sx={{ overflowX: 'hidden', padding: 0 }}>
+                    <Form $gap={0}>
+                        <Grid container spacing={2}>
+                            <Grid item xs={12}>
+                                <Typography variant="h5">{translate('Credential.GeneralInfo.Title')}</Typography>
+                            </Grid>
+                            <Grid item xs={12}>
+                                <TextField
+                                    fullWidth
+                                    label={translate('Credential.Details.Name.Label')}
+                                    required
+                                    name="name"
+                                    InputLabelProps={{ shrink: true }}
+                                    value={credentialFormState.name}
+                                    onChange={handleInputChange}
+                                    error={!isFormValid}
+                                    helperText={!isFormValid && inputErrorMessages.NAME_IS_REQUIRED}
+                                ></TextField>
+                            </Grid>
+                            <Grid item xs={12}>
+                                <TextField
+                                    fullWidth
+                                    label={translate('Credential.Details.Description.Label')}
+                                    multiline
+                                    name="description"
```

Comment:
should use translation too

Reply:
what should use the translation? label has it, name shouldn't

Reply:
what is it then?

Reply:
the 'name' attribute